### ISSUE TITLE
Used Electron Version uses vulnerable Chromium

### ISSUE THREAD

**Issue by nhaug** [microsoft/vscode#228445]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.93.1
- OS Version: macOs Sonoma 14.6.1 (23G93)

Steps to Reproduce:

1. Create a Proxy Rule, which is not allowing chrome v. 120 - 126 because of known vulnerabilities
2. Try to Search for Extension -> Error Message is shown "unable to fetch"
3. In the Proxy Log I see an old Chromium Version

At first I opened a security Issue at Microsoft, but it was sadly closed without giving me a hint, were i should report this issue.

See also:

August 22, 2024
Microsoft has a fix for [CVE-2024-7971](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-7971) and [CVE-2024-7965](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-7965) to Microsoft Edge Stable Channel (Version 128.0.2739.42) which have been reported by the Chromium team as having an exploit in the wild. For more information, see the [Security Update Guide](https://msrc.microsoft.com/update-guide).
 Note
It's worth highlighting that [Microsoft Edge's enhanced security mode](https://learn.microsoft.com/en-us/DeployEdge/microsoft-edge-security-browse-safer) feature mitigates the vulnerability described in [CVE-2024-7971](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-7971). You can opt-in into this security feature and have peace of mind that Microsoft Edge is protecting you against this exploit.

**Comment by nhaug** [comment]
I have to correct : versions 120 - 127 are affected

**Comment by deepak1556** [comment]
We do security patch releases only in case we become aware of a vulnerability that affects VS Code. We are tracking all CVEs issued for all of our dependencies, including node.js, v8, chromium, electron, etc, and we have a [mechanism to receive VS Code specific security issues](https://github.com/microsoft/vscode/blob/main/SECURITY.md#reporting-security-issues).

Finally, we update regularly (monthly) our dependencies, Chromium 128 is being tracked in https://github.com/microsoft/vscode/pull/218245 and it will be available with insiders starting next month.

**TL;DR** Your security scan is correct, we include Chromium 124, but that does not mean your system is in any way vulnerable to those CVEs because we use Chromium via Electron in a very specific way.
